src/dfmt/editorconfig.d

```diff
@@ -43,7 +43,8 @@ enum EOL : ubyte
     unspecified,
     lf,
     cr,
-    crlf
+    crlf,
+    automatic
```

Comment:
can we rename this to `_automatic` if it's just an internal value to make sure people don't use it?

Reply:
My first take on this was indeed using `unspecified`, but that messed with line length calculations, where a line ending in `\r\n` started to break differently from the same line when it ended in `\n`.

`unspecified` is used when merging two configurations. By adding an additional enum member the line length issues disappeared.

Reply:
ok, makes sense. Yes I think also renaming it to _unspecified makes sense.